### PR TITLE
Update four_layers.md

### PR DESCRIPTION
**Pull request by arthuo**
Fix some Chinese translations in the documentation

### CODE REVIEW

File: site/zh-CN/reference/architecture/four_layers.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Index node 负责执行索引构建任务。Index node不需要常驻于内存
 
 ### 元数据存储
 
-负责存储元信息的快照，比如 collection schema 信息、节点状态信息、消息消费的 checkpoint 等。元信息存储需要极高的可用性、强一致和事务支持，因此 etcd 是这个场景下的不二选择。除此之外，etcd 还承担了服务注册和健康检查的职责。
+负责存储元信息的快照，比如 collection schema 信息、节点状态信息、消息使用的 checkpoint 等。元信息存储需要极高的可用性、强一致和事务支持，因此 etcd 是这个场景下的不二选择。除此之外，etcd 还承担了服务注册和健康检查的职责。
```
Comment (LocoRichard) [inline site/zh-CN/reference/architecture/four_layers.md:60]: I think the original verb is correct. Usually we say that a node consumes messages from a certain checkpoint.

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by LocoRichard** [comment]
Hi @arthuo , thank you for correction, I have made my feedback. And one more thing is that, we are collecting the contacts of active contributors in our community for event notification, swag shipping, and stuff. Would you to share your name, Wechat ID, Email, Company, Location (city), Mailing Address, and T-shirt Size by email the information to `lichen.wang@zilliz.com`. I would only share these information with our community manager. You don't have to share all of them with us if you don't want to. And it's totally ok if you don't want to share it at all.

File: site/zh-CN/reference/architecture/four_layers.md (hunk 1 above)
Comment (arthuo) [reply site/zh-CN/reference/architecture/four_layers.md]: > I think the original verb is correct. Usually we say that a node consumes messages from a certain checkpoint.

Thank you for your reply, I have learned a lot.

**Comment by arthuo** [comment]
> 



> Hi @arthuo , thank you for correction, I have made my feedback. And one more thing is that, we are collecting the contact information of active contributors in our community. Would you mind sharing your name, Wechat ID, Email, Company/Institute, Location (city), Mailing Address, and T-shirt Size by email the information to `lichen.wang@zilliz.com`? I would only share these information with our community manager. You don't have to share all of them with us if you don't want to. And it's totally ok if you don't feel like it at all. We would only use these information for event notification, swag shipping, and stuff.

Thanks, I will send an email later.

**Pull request merged**